.github/issue_template/sitemap_change.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Suggest a Sitemap change
+about: 'Suggest a change in the Sitemap syntax and corresponding UIs'
+labels: sitemap
+---
+
+### Is your feature request related to a problem? Please describe.
+<!--
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+-->
+
+### Describe the solution you'd like
+<!--
+A clear and concise description of what you want to happen.
+-->
+
+### Describe alternatives you've considered
+<!--
+A clear and concise description of any alternative solutions or features you've considered.
+-->
+
+### Additional context
+<!--
+Add any other context or screenshots about the feature request here.
+-->
+
+### Coordination between maintainers
+<!--
+Don't edit or remove this section.
+-->
+
+Notify maintainers of other UIs:
+@openhab/webui-maintainers 
+@openhab/android-maintainers 
+@openhab/ios-maintainers 
+
+Checklist for implementation:
+- [ ] Core
+- [ ] Basic UI
+- [ ] Android app
+- [ ] iOS app
+- [ ] Docs
```

Comment:
Main UI should also be added to the list, because you can use it to create sitemaps.

Reply:
Done